.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+Thanks for submitting an issue! 
+
+Please answer these questions
+
+-------------ANY REQUEST NEEDS THE FOLLOWING --------------------
+
+I. What keywords did you search in Kubernetes issues before filing this one? (If you have found any duplicates, you should instead reply there.):
```

Comment:
Few suggestions here

1. We should reference the kops issue tracker, instead of the Kubernetes issue tracker
2. The issue tracker referenced should be a link
3. We should only have parenthetical sentences if the other content reads well without it. Here we should just have another sentence.
4. We should encourage users to reply to other issues, or at least point them to the new issue. Sometimes they are very close, but slightly different.